### DISCUSSION TITLE
addPullRequestReviewThreadReply is missing

### DISCUSSION THREAD

**Discussion by aleppard** [community/community#59924] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Hello,

The mutation addPullRequestReviewComment is being deprecated on 2023-10-01. In the documentation it says to use  addPullRequestReviewThread or addPullRequestReviewThreadReply instead. However, addPullRequestReviewThreadReply does not exist and you can't specify the "in reply to" field for addPullRequestReviewThread.

With the GraphQL is there any way to reply to a comment thread (including in a review) using a non-deprecated method? Thanks.

See https://docs.github.com/en/graphql/reference/input-objects#addpullrequestreviewcommentinput

> Upcoming Change on 2023-10-01 
> UTC Description: body will be removed. 
> use addPullRequestReviewThread or addPullRequestReviewThreadReply instead Reason: We are deprecating the > addPullRequestReviewComment mutation.

**Comment by aleppard** [comment]
The REST API also does not appear to have any way of replying to a comment in a pull request review.

**Reply by dbalatero** [reply]
@aleppard The REST API does let you reply to a comment:

```
POST https://api.github.com/repos/your-org/your-repo/pulls/123/comments/1276685280/replies

{ "body": "The reply you want to make to comment ID #1276685280" }
```

However, this requires you to have the older `int`-based comment ID to construct the above URL. If you have the newer GraphQL style ID, you'll have to exchange it for the `int` ID first before you can use this API, which adds another trip to Github (500-1200ms in my average testing).

**Comment by dbalatero** [comment]
For now, we're going to use the deprecated `addPullRequestReviewComment` mutation, but I'd also really like to know where the `addPullRequestReviewThreadReply` mentioned is? It's pretty scary to rely on any of these APIs when the migration path for an API being deprecated in 2-3 months doesn't exist.

**Comment by dbalatero** [comment]
@aleppard After some support ticket back and forth the new API seems to be available now:

https://docs.github.com/en/graphql/reference/input-objects#addpullrequestreviewthreadreplyinput

so you should be good to move over to the new API now!

**Reply by aleppard** [reply]
Awesome, thanks for letting me know!
